src/actions/continuous-deploy-fingerprint.ts

```diff
@@ -41,40 +43,48 @@ export async function continuousDeployFingerprintAction(input = collectContinuou
 
   info('Looking for builds with matching runtime version (fingerprint)...');
 
-  let androidBuildInfo = await getBuildInfoWithFingerprintAsync({
+  const existingAndroidBuildInfo = await getBuildInfoWithFingerprintAsync({
     cwd: input.workingDirectory,
     platform: 'android',
     profile: input.profile,
     fingerprintHash: androidFingerprintHash,
     excludeExpiredBuilds: isInPullRequest,
   });
-  if (androidBuildInfo) {
-    info(`Existing Android build found with matching fingerprint: ${androidBuildInfo.id}`);
+  let androidBuildRunInfo: BuildRunInfo;
+  if (existingAndroidBuildInfo) {
+    info(`Existing Android build found with matching fingerprint: ${existingAndroidBuildInfo.id}`);
+    androidBuildRunInfo = { buildInfo: existingAndroidBuildInfo, isNew: false };
   } else {
     info(`No existing Android build found for fingerprint, starting a new build...`);
-    androidBuildInfo = await createEASBuildAsync({
-      cwd: input.workingDirectory,
-      platform: 'android',
-      profile: input.profile,
-    });
+    androidBuildRunInfo = {
+      buildInfo: await createEASBuildAsync({
+        cwd: input.workingDirectory,
+        platform: 'android',
+        profile: input.profile,
+      }),
+      isNew: true,
+    };
   }
 
-  let iosBuildInfo = await getBuildInfoWithFingerprintAsync({
+  const existingIosBuildInfo = await getBuildInfoWithFingerprintAsync({
     cwd: input.workingDirectory,
     platform: 'ios',
     profile: input.profile,
     fingerprintHash: iosFingerprintHash,
     excludeExpiredBuilds: isInPullRequest,
   });
-  if (iosBuildInfo) {
-    info(`Existing iOS build found with matching fingerprint: ${iosBuildInfo.id}`);
+  let iosBuildRunInfo: BuildRunInfo;
+  if (existingIosBuildInfo) {
+    info(`Existing iOS build found with matching fingerprint: ${existingIosBuildInfo.id}`);
+    iosBuildRunInfo = { buildInfo: existingIosBuildInfo, isNew: false };
   } else {
     info(`No existing iOS build found for fingerprint, starting a new build...`);
-    iosBuildInfo = await createEASBuildAsync({ cwd: input.workingDirectory, platform: 'ios', profile: input.profile });
+    iosBuildRunInfo = {
+      buildInfo: await createEASBuildAsync({ cwd: input.workingDirectory, platform: 'ios', profile: input.profile }),
+      isNew: true,
+    };
```

Comment:
```suggestion
    iosBuildRunInfo = {
      buildInfo: await createEASBuildAsync({
        cwd: input.workingDirectory,
        platform: 'ios',
        profile: input.profile,
      }),
      isNew: true,
    };
```

nit: indent as android